frontend/config-core_demo.json

```diff
@@ -1,6 +1,6 @@
 {
-  "@openmrs/esm-styleguide": {
-    "Brand color #1": "#005d5d"
+  "@openmrs/esm-patient-chart-app": {
+    "showUpcomingAppointments": true
```

Comment:
enabling this configures the visit note so that it will show upcoming appointments when entering a visit note.  Seemed like it would make sense to turn this on in the refapp for demo purposes?  And this seems like the proper place to configure the refapp?